warema_wms/wms_controller.py

```diff
@@ -18,16 +18,17 @@
 RX_SHADE_STATE = '0431'
 
 TX_MOVE_SHADE = '0821'
-SHADE_POSITION = '03{}ffffff'
+SHADE_POSITION = '03{}dfffff'   # df = trial to move raffstores in tilt closed pos., working in console but not in HA
+TILT_POSITION = '03ff{}ffff'
+STOP_SHADE = '01ffffffff'       # at least at my setup it works (instead of STOP_SHADE = '0001ffffff')
```

Comment:
```suggestion
STOP_SHADE = '01ffffffff'
```

I believe you. I never actually tried the stop feature out, since I don't need it.

Reply:
i'm really not sure anymore, but i think it had worked with your command as i tried your feat_stop_branch some month ago, but I knowingly didn't update the web control box or so... 
this '01ffffffff' string i found with wireshark and as i commented, at least at my setup it works with my 6 venetian blinds (which are two different types of)

the stop command was my entrypoint into my "reverse engineering" cause it was the easiest way for me to get into it... 
actually I don't need it either, haha